modules/ROOT/pages/index.adoc

```diff
@@ -75,7 +68,26 @@ run them with Kubernetes across many cloud platforms.
 === Other Active Projects
 
 * link:https://github.com/neo4j-contrib/neovis.js/[Neovis.js^], a graph visualization toolkit for the web
-* link:https://github.com/neo4j-contrib/neo4j-spark-connector/[The Neo4j-Spark-Connector^], library to exchange graph data between Neo4j and Apache Spark
+
+== Graduated Projects
+
```

Comment:
Maybe a sentence here that describes what it means to graduate. Something like "The following projects were successfully developed, incubated, and validated within Neo4j Labs and have graduated to official and supported Neo4j products ready for production use at scale."